ipa.go

```diff
@@ -60,3 +60,9 @@ func FromBytes(fr *Fr, data []byte) {
 	copy(aligned[32-len(data):], data)
 	fr.SetBytes(aligned[:])
 }
+
+func MapToScalarFieldBytes(point *Point) [32]byte {
```

Comment:
Whether it should go into go-ipa can be bikeshedded imo -- main goal is to update all libraries to have this breaking change